### PR TITLE
Milestone: To Do list: interactive list

### PR DESCRIPTION
**Pull request by kendoriddy**
For this milestone, I implemented the following

- Add a new JavaScript file and import it as a module (it contain methods related to the status updates (completed: true / false)).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

Here is a video demo of the app
https://www.loom.com/share/6ddf6015b06948108aad29c642d0f804

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -99,31 +101,37 @@ const showToDos = () => {
...
 
+    document.querySelector('#clear-completed').style.display = 'block';
+
     document.querySelectorAll('.edit').forEach((e) => {
       e.addEventListener('click', editTodos);
     });
+
+    document.querySelectorAll('.check').forEach((e) => {
+      e.addEventListener('change', completedTodos);
+    });
```
Comment (EricMbouwe) [inline src/crud.js:154]: - [ ]  Your tasks list is not keeping the localStorage state after a page refresh. Completed tasks are not checked anymore meanwhile their status is true. You should probably check your getList function to fix that.

onchange2

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @kendoriddy, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- Linters ✔
- PR ✔
- Styles ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js [hunk 2]
```diff
@@ -99,36 +108,73 @@ const showToDos = () => {
...
+      ['focus', 'blur', 'keyup'].forEach((evt) => {
+        input.addEventListener(evt, (e) => {
+          if (evt === 'blur' || (evt === 'keyup' && e.key === 'Enter')) {
+          // It's a modify
+            const taskIndex = Number(e.target.id.split('-')[1]);
+            saveTodo(taskIndex);
+            e.target.blur();
+          }
+        });
+      });
```
Comment (BertrandConxy) [inline src/crud.js]: - [ ] Good job so far. I can see that you made changes based on the previous reviewer's suggestions. However, the changes you introduced may have caused some bugs in your application. **Right now, when I run the application locally, I tried to add a task and when I click `enter` the page reloaded and the task is neither displayed in the UI nor stored in the local storage.** Please visit back your function to add a task and check if it is working as before 🙏 As a result, I was not able to check whether the completed functionality is working.

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @kendoriddy , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 💪 
- no linter errors
- correct flow is used
- good PR


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js (hunk 2 above)
Comment (kendoriddy) [reply src/crud.js]: @BertrandConxy Thanks for your feedback. However, I am able to add new tasks on my local machine without any issue. Can you please try it again? I will add a video to demonstrate functionalities too

Comment (kendoriddy) [reply src/crud.js]: @BertrandConxy here is a video demo of the app
https://www.loom.com/share/6ddf6015b06948108aad29c642d0f804

Comment (BertrandConxy) [reply src/crud.js]: Ok. I can see it. It looks like we even have a different UI. Let me review it again

**Review by BertrandConxy — APPROVED** [review]
Hi @kendoriddy , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- no linter errors
- good PR
- good README

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**